specification/communication/data-plane/Microsoft.CommunicationServicesTurn/preview/2021-02-22-preview1/CommunicationTurn.json

```diff
@@ -0,0 +1,180 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
```

Comment:
This is the same title from `CommunicationIdentity.json` swagger.  Is this copy/paste mistake?

Reply:
Not a mistake, just wasn't sure the purpose of the title and I was intending this to be under the same namespace when in an SDK, but I can rename it to `CommuncationTurn`